processors/blurc/process.c

```diff
@@ -0,0 +1,18 @@
+#include <string.h>
+#include <wasmcv/imports.h>
+
+
+wasm_cv_mat_borrow_mat_t process(wasm_cv_mat_borrow_mat_t image) {
+    wasm_cv_cv_own_mat_t frame;
+    wasm_cv_cv_own_mat_t out_mat;
+    wasm_cv_cv_size_t size = {.x = 25, .y = 25};
+       
+    frame = wasm_cv_mat_method_mat_clone(image);
```

Comment:
Is these two lines actually needed, or can you just call 

```c
out_mat = wasm_cv_cv_blur(image, &size);
```

Reply:
I was playing with it and I now understand why you did it. Never mind my comment!